source/whats-new.txt

```diff
@@ -48,55 +48,11 @@ Learn what's new in:
 What's New in 6.0
 -----------------
 
-.. important:: Breaking Changes in v6.0
+.. warning:: Breaking Changes in v6.0
 
-   - Version 6.0 of the {+driver-short+} requires Node.js v16.20.1 or later.
-
-   - If you add ``mongodb-client-encryption`` as a dependency,
-     the major version number must match that of the {+driver-short+}. For example,
-     {+driver-short+} v6.x.x requires ``mongodb-client-encryption`` v6.x.x.
-
-   - Automatic Encryption methods are now in the {+driver-short+}. You must
-     import these methods from the driver instead of from
-     ``mongodb-client-encryption``.
-
-   - Removed the ``ObjectId`` constructor that accepted a 12-character string.
-
-   - Modified ``abortTransaction()`` and ``commitTransaction()`` methods to return
-     ``null`` instead of the raw command results.
-
-   - Removed connection option helpers that accepted values other than ``true``
-     or ``false`` as booleans. You must provide either ``true`` or ``false`` values in
-     the connection string or to the MongoClient constructor.
-
-   - Removed the ``Binary`` BSON type constructor that accepted a string.
-
-   - The ``Binary.write()`` method no longer accepts a string to write
-     to the binary BSON object.
-
-   - The ClientEncryption API returns promises instead of callbacks.
-
-   - The ``socks`` package, which enables SOCKS5 proxy support, is a
-     peer-optional dependency. To use ``socks`` functionality in your
-     application, you must install the package. To learn more, see
-     :ref:`node-connect-socks`.
-     
-   - If you start a session on a client, then pass that session to a
-     different client, the driver throws an error when you
-     perform any operations in the session.
-   
-   - The ``includeResultMetadata`` option for compound operation methods
-     is ``false`` by default. See the :ref:`Built-in
-     Methods <node-compound-operations-builtin>` section of the Compound
-     Operations guide for more information.
```

Comment:
S: for certain entries, a page is linked in this note, but not in the breaking changes guide. Could you go through and make sure that additional information is linked in the breaking changes guide wherever they have been linked in these admonitions?

Reply:
Thanks for catching this! I've added the links to installing the socks package (node-connect-socks) and information about compound operations (node-compound-operations-builtin).